lib/util/tooltip.coffee

```diff
@@ -0,0 +1,54 @@
+# InkTooltip
+#
+# Tooltips that support arbitrary HTML content.
+#
+# new InkTooltip(parent, content, cond = -> true)
+#     Create a new InkTooltip attached to `parent` with content `content`. Will
+#     be shown on mouseover, unless `cond` evaluates to `false`.
+#
+# .show()
+#     Show the tooltip.
+#
+# .hide()
+#     Hide the tooltip.
+#
+# .destroy()
+#     Destroy the tooltip.
+
+module.exports =
+class InkTooltip
```

Comment:
It seems redundant to refer to this as `InkTooltip`; outside of the package it'll be called `ink.Tooltip` anyway.